parseAPI/src/Parser.C

```diff
@@ -101,12 +101,12 @@ Parser::Parser(CodeObject & obj, CFGFactory & fact, ParseCallbackManager & pcb)
         plt_entries[lit->first] = lit->second;
     }
 
-    if(obj.cs()->regions().empty()) {
+    if(obj.cs()->regions().empty() ||
+      (obj.cs()->regions()[0] == NULL && obj.cs()->regions().size() == 1)) {
```

Comment:
By default, a `StandardParseData` parser must initialize its `_rdata` member with the first element in code regions. Thus, I pushed a NULL region into the code region just to avoid getting crashed and still want an `UNPARSEABLE` parser.

```
StandardParseData::StandardParseData(Parser *p) :
ParseData(p), _rdata(&p->obj(), p->obj().cs()->regions()[0])
{ }
```

Reply:
Instead of adding the NULL region externally, maybe we can add it here when we see `obj.cs()->regions().empty()`? @mxz297 , what is your suggestion on this issue?

Reply:
The patch I attached below does what you suggested.